src/core/ui/about.cpp

```diff
@@ -137,28 +136,11 @@ QString AboutDialog::tabHelpUs()
 QString AboutDialog::tabThanks()
 {
     QString str;
-    str += QLatin1String("<b>") + tr("Translate:") + QLatin1String("</b>");
-    str += QLatin1String("<br>");
-    str += tr(" Brazilian Portuguese translation")  + QLatin1String("<br>");
-    str += tr("Marcio Moraes") + QLatin1String(" &lt;marciopanto@gmail.com&gt;<br>");
-    str += QLatin1String("<br>");
-    str += tr(" Ukrainian translation") + QLatin1String("<br>");
-    str += tr("Gennadi Motsyo") + QLatin1String(" &lt;drool@altlinux.ru&gt;<br>");
-    str += QLatin1String("<br>");
-    str += tr(" Spanish translation") + QLatin1String("<br>");
-    str += tr("Burjans L García D") + QLatin1String(" &lt;burjans@gmail.com&gt;<br>");
-    str += QLatin1String("<br>");
-    str += tr(" Italian translation") + QLatin1String("<br>");
-    str += QLatin1String("speps &lt;dreamspepser@yahoo.it&gt;<br>");
-    str += QLatin1String("<br>");
 
-    str += QLatin1String("<b>") + tr("Testing:") + QLatin1String("</b>");
-    str += QLatin1String("<br>");
-    str += QLatin1String("Jerome Leclanche - ") + tr("Dual monitor support and other in Linux") + QLatin1String("<br>");
-    str += QLatin1String("Alexander Sokolov - ") + tr("Dual monitor support in Linux") + QLatin1String("<br>");
-    str += QLatin1String("Alexantia - ") + tr("win32-build [Windows XP and 7]") + QLatin1String("<br>");
-    str += QLatin1String("iNight - ") + tr("old win32-build [Windows Vista]") + QLatin1String("<br>");
-    str += QLatin1String("burjans - ") + tr("win32-build [Windows 7]") + QLatin1String("<br>");
+    str += QLatin1String("<b>") + tr("Special thanks to:") + QLatin1String("</b>");
+    str += QLatin1String("<ul>");
+    str += QLatin1String("<li>") + QLatin1String("Artem Galichkin - DOOMer at GitHub (") + tr("for creating ScreenGrab") + QLatin1String(")</li>");
+    str += QLatin1String("<li>") + QLatin1String("Marcus Britanicus at GitHub (") + tr("for supporting wlr screencopy protocol") + QLatin1String(")</li>");
```

Comment:
Adding the "translators team" of weblate?

Reply:
"Special thanks" can't be so general, by definition ;)